plugins/dell/fu-plugin-dell.c

```diff
@@ -310,15 +315,15 @@ fu_plugin_dell_capsule_supported (FuPlugin *plugin)
 }
 
 static gboolean
-fu_plugin_dock_node (FuPlugin *plugin, GUsbDevice *device,
-		     guint8 type, const gchar *component_guid,
-		     const gchar *component_desc, const gchar *version)
+fu_plugin_dock_node (FuPlugin *plugin, GUsbDevice *device, guint8 type,
+		     const gchar *component_guid, const gchar *component_desc,
+		     const gchar *version)
 {
 	const gchar *dock_type;
 	g_autofree gchar *dock_id = NULL;
 	g_autofree gchar *dock_key = NULL;
 	g_autofree gchar *dock_name = NULL;
-	g_autoptr(FuDevice) dev = NULL;
+	g_autoptr (FuDevice) dev = NULL;
```

Comment:
This is also a pre-processor thing -- the "type" can be thought of as `g_autoptr(FuDevice)` rather than `g_autoptr` being that looks like a function on a type.

Reply:
Ah this is what @YehezkelShB was referring to when we were talking about parentheses.  I  think this will be difficult to adjust this one unfortunately.

Reply:
corrected